src/main/groovy/com/vaadin/gradle/VaadinPlugin.groovy

```diff
@@ -109,7 +109,12 @@ class VaadinPlugin implements Plugin<Project> {
                     create(VaadinClientDependenciesExtension.NAME, VaadinClientDependenciesExtension, project)
                 } else {
                     // need to use reflection since Groovy+Kotlin projects are not really supported.
-                    create('vaadin', Class.forName("com.vaadin.gradle.VaadinFlowPluginExtension"), project)
+                    def extensionName = 'vaadin'
+                    if (findByName(extensionName) != null) {
+                        // fixes https://github.com/vaadin/vaadin-gradle-plugin/issues/26
+                        extensionName = 'vaadinFlow'
```

Comment:
I think this is the only way to do it, chatted with Lois from Gradle team. But let's use different name, lets use vaadin14 if there is a collision.